src/screens/SUE/SueListScreen.tsx

```diff
@@ -81,7 +107,23 @@ export const SueListScreen = ({ navigation, route }: Props) => {
   const listItems = useMemo(() => {
     if (!data?.length) return [];
 
-    return parseListItemsFromQuery(query, data, undefined, { appDesignSystem }).reverse();
+    let parsedListItem = parseListItemsFromQuery(query, data, undefined, {
+      appDesignSystem
+    }).reverse();
```

Comment:
let's reverse at the end. if we have a `sortBy` set, we do not need to `reverse` the values here previously

Reply:
But then the list we sorted with `sortBy` will be reversed. Do we want this? For example, if we sort by creation date, the oldest created item will be at the top of the list.

Reply:
I updated the sort property in the code with `_sortBy` and added a new control for the reverse property

https://github.com/smart-village-solutions/smart-village-app-app/pull/1030/commits/3280b8268bdc6a5832f8b13dc5473dc2e625bd80

Reply:
ok true, i thought you were returning early, so that the reverse would have no effect.
well done updating the sorts, thanks.